From ec7ff1612b2f5b0075545dc705b7c2610ec83748 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Fri, 10 Feb 2023 11:43:06 +0100
Subject: [PATCH 2/2] rh openssl: add crypto/bn/rsa_sup_mul.c to file list

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
RH-MergeRequest: 21: openssl update
RH-Bugzilla: 2164531 2164543 2164558 2164581
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Commit: [2/2] 61acf48e337f04b34c4f309241775b204ae2e54f (kraxel/rhel-edk-2)
---
 CryptoPkg/Library/OpensslLib/OpensslLib.inf       | 1 +
 CryptoPkg/Library/OpensslLib/OpensslLibCrypto.inf | 1 +
 2 files changed, 2 insertions(+)

diff --git a/CryptoPkg/Library/OpensslLib/OpensslLib.inf b/CryptoPkg/Library/OpensslLib/OpensslLib.inf
index 19913a4ac6..4eaa8a756d 100644
--- a/CryptoPkg/Library/OpensslLib/OpensslLib.inf
+++ b/CryptoPkg/Library/OpensslLib/OpensslLib.inf
@@ -571,6 +571,7 @@
   $(OPENSSL_PATH)/ssl/statem/statem_local.h
 # Autogenerated files list ends here
 # RHEL8-specific OpenSSL file list starts here
+  $(OPENSSL_PATH)/crypto/bn/rsa_sup_mul.c
   $(OPENSSL_PATH)/crypto/evp/kdf_lib.c
   $(OPENSSL_PATH)/crypto/evp/pkey_kdf.c
   $(OPENSSL_PATH)/crypto/kdf/kbkdf.c
diff --git a/CryptoPkg/Library/OpensslLib/OpensslLibCrypto.inf b/CryptoPkg/Library/OpensslLib/OpensslLibCrypto.inf
index 5057857e8d..eec4771f2c 100644
--- a/CryptoPkg/Library/OpensslLib/OpensslLibCrypto.inf
+++ b/CryptoPkg/Library/OpensslLib/OpensslLibCrypto.inf
@@ -520,6 +520,7 @@
   $(OPENSSL_PATH)/crypto/x509v3/v3_admis.h
 # Autogenerated files list ends here
 # RHEL8-specific OpenSSL file list starts here
+  $(OPENSSL_PATH)/crypto/bn/rsa_sup_mul.c
   $(OPENSSL_PATH)/crypto/evp/kdf_lib.c
   $(OPENSSL_PATH)/crypto/evp/pkey_kdf.c
   $(OPENSSL_PATH)/crypto/kdf/kbkdf.c
-- 
2.37.3

